### ISSUE TITLE
add missing functionality for nomad units in blueprint.lua

### ISSUE THREAD

**Issue by Exotic-Retard** [Nomads-Project/nomads#29]
first thing is to have the cloakmeshblueprint extracted for them too. 
then probably a bunch of other stuff like mod gsr and what not, then wrecks has some dubious code going on in there too, bscly need to slough through it again and refactor

**Comment by CookieNoob** [comment]
nomads dont have any cloaked units...

what else needs to be added?

**Comment by Exotic-Retard** [comment]
the cloakmesh blueprint should be done anyway because of mod compatibility with things like blops, which have aoe cloak. its done for all fa units so it should be done for all nomads units too.

modding GSR is the guardscanradius which makes attack move better for long range units. tldr everything should stop on its max radius.

then theres sacu presets, which might be removable now that its in faf.

in short, blueprints.lua is particularly sensitive and needs refactoring so that nomads get the same treatment faf units do, and with minimum destructive hooking.

**Comment by CookieNoob** [comment]
I think this functionality is implemented in FAF with the latest patch, isnt it? @Exotic-Retard so we can close this now?

The only thing that should be done with blueprints.lua is checking if maybe some functions that mod the blueprints slow down the start of the lobby significantly.
